src/sql/parts/notebook/cellViews/code.component.ts

```diff
@@ -177,14 +179,16 @@ export class CodeComponent extends AngularDisposable implements OnInit, OnChange
 
 		this._register(this._editor);
 		this._register(this._editorInput);
-		this._register(this._editorModel.onDidChangeContent(e => {
-			this._editor.setHeightToScrollHeight();
-			this.cellModel.source = this._editorModel.getValue();
-			this.onContentChanged.emit();
-			this.checkForLanguageMagics();
-			// TODO see if there's a better way to handle reassessing size.
-			setTimeout(() => this._layoutEmitter.fire(), 250);
-		}));
+		if (this._editorModel) {
```

Comment:
If not, should we somehow cleanly error? We basically can't function at this point.